dist/backdraft.js

```diff
@@ -1214,42 +1258,68 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
+      var table = this;
+      table.activeMenu = null;
       var filter = col.filter;
+      var filterActive = filter.value || filter.eq || filter.gt || filter.lt;
```

Comment:
related to above.. is this the active filter, or just a boolean? I think the latter.. so how about isFilterActive ?